src/pybind/mgr/zabbix/module.py

```diff
@@ -257,13 +260,13 @@ def serve(self):
         while self.run:
             self.log.debug('Waking up for new iteration')
 
-            # Sometimes fetching data fails, should be fixed by PR #16020
             try:
                 self.send()
             except Exception as exc:
-                self.log.error(exc)
+                # Shouldn't happen, but let's log it and retry next interval,
+                # rather than dying completely.
+                self.log.exception("Unexpected error during send():")
 
-            interval = self.config['interval']
```

Comment:
Why remove this line? Because it will not have the interval value available this way?

Reply:
Oops, that was just a typo, will fix